Jenkinsfile

```diff
@@ -6,14 +8,17 @@ pipeline {
   }
   stages {
     stage('Test') {
+      environment {
+         IS_BETA = "${env.CHANGE_ID && pullRequest.labels.contains('Beta') ? true : false}"
```

Comment:
Note I had to install [this plugin](https://github.com/jenkinsci/pipeline-github-plugin) on Jenkins for this to work.